### PR TITLE
Add scopes field to integration structure

### LINKED ISSUE #2406: [api-docs] Add `scopes` field to integration structure (#5311)
A new commit was made into the api-docs repo: https://github.com/discord/discord-api-docs/commit/03ba219d490701352ed1d5b9307b8199e458641c
No details given.

This is a bot created issue.

### PR DESCRIPTION
**Pull request by lts20050703**
closes #2406 
upstream https://github.com/discord/discord-api-docs/commit/03ba219d490701352ed1d5b9307b8199e458641c

### CODE REVIEW

File: types/discord.ts [hunk 1]
```diff
@@ -134,6 +133,8 @@ export interface DiscordIntegration {
   account: DiscordIntegrationAccount;
   /** The bot/OAuth2 application for discord integrations */
   application?: DiscordIntegrationApplication;
+  /** the scopes the application has been authorized for */
+  scopes: string[];
```
Comment (Skillz4Killz) [inline types/discord.ts:137]: No changes in transformer or helper?

File: types/discord.ts [hunk 2]
```diff
@@ -17,7 +17,6 @@ import {
   GuildNsfwLevel,
   IntegrationExpireBehaviors,
   InteractionTypes,
-  Locales,
```
Comment (Skillz4Killz) [inline types/discord.ts:20]: intentional?

Comment (lts20050703) [reply types/discord.ts]: yes. it's unused so it's removed. should i keep that unrelated changes or not

Comment (lts20050703) [reply types/discord.ts]: actually i just merge main into this branch and it's already removed?

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**